openlane/scripts/klayout/xor.drc

```diff
@@ -83,14 +99,14 @@ ignore_list = $ignore.split(";")
 
 ## Perform per-layer XOR
 total_xor_differences = 0
-layers.keys.sort.each do |layer_name|
+layers.sort.uniq.each do |layer_name, layer_data|
   if ignore_list.include? layer_name
-    warn "Skipping #{layer_name}"
+    warn "--- Skipping #{layer_name}… ---"
   else
     info "--- Running XOR for layer #{layer_name} ---"
 
     layer_info = layers[layer_name]
-    xor_data = a.input(layer_name) ^ b.input(layer_name)
+    xor_data = (a.input(layer_name, layer_data) ^ b.input(layer_name, layer_data))
```

Comment:
I don't think this is needed.

Reply:
I tried to match the KLayout XOR script written by Mathias a bit better.